pkg/notify/subscriptionnotification.go

```diff
@@ -195,12 +199,19 @@ func (s *SubscriptionNotification) BuildSMTPMessage(template *config.EmailTempla
 
 	log.Debugf("Sending email %s, %s, %s", fromAddress, toAddress, subject)
 
+	emailBody := s.UpdateTemplate(template.Body)
+	if strings.Contains(template.Body, "$") {
```

Comment:
should this be "${" to ensure its not just part of the value?

Reply:
true... will adjust.

Reply:
is it deprecated or removed?  I do not see how it is still using the ${

Reply:
technically, to be removed, i guess... maybe i change the warning... 
line 202 still uses the '${' (tag)... so the template is updated there.  If it does contain '${', i give them a warning its going to go away...  otherwise, I set the emailBody on 206 using the go template way.

maybe I need to put better checking around which kind of tag they are using.

Reply:
so UpdateTemplate will replace the ${var} with {{ .Var }} until removed?

Reply:
Oh, I see, you are running one or the other.

Reply:
Maybe you can check if emailBody is equivalent to template.Body to raise this warn?

Reply:
For now yes... Once we are sure we can do away with ${ , I will clean this up better.